src/app/lib/button/button.component.scss

```diff
@@ -1,618 +1,137 @@
 @import '../colors.scss';
```

Comment:
This is amazing how you condensed all of the styles into a reusable and maintainable file. Great work!